packages/react-components/src/organisms/MainNavigation.tsx

```diff
@@ -51,6 +51,9 @@ const listStyles = css({
     12 / perRem,
     'em',
   )})`,
+  li: {
```

Comment:
We try to avoid selecting other elements from parent elements to encapsulate their logic a notable exception being SVGs. Going to approve this as its only targeting things in the same file so its still encapsulated